rsconnect/bundle.py

```diff
@@ -642,7 +642,9 @@ def make_html_bundle_content(
     return manifest, relevant_files
 
 
-def infer_entrypoint(path, mimetype):
+def infer_entrypoint(*args, **kwargs):
```

Comment:
Where we have specific, known arguments, we should keep those (and add types) rather than using * or **.